frontend/components/events/EventPayment.tsx

```diff
@@ -157,3 +143,49 @@ export default function EventPayment(props: EventPaymentProps) {
     </div>
   );
 }
+
+const SameDayEventDateTime = ({ startDate, endDate }: { startDate: Timestamp; endDate: Timestamp }) => {
+  const { hours, minutes } = duration(startDate, endDate);
+  return (
+    <>
+      <h2 className=" font-semibold">Date and Time</h2>
+      <div className="flex items-center">
+        <CalendarDaysIcon className="w-5 mr-2" />
+        <p className="text-md mr-[5%] font-light">{timestampToDateString(startDate)}</p>
+      </div>
+      <div className="flex items-center">
+        <ClockIcon className="w-5 mr-2" />
+        <p className="text-md mr-[5%] font-light">
+          {timestampToTimeOfDay(startDate)} - {timestampToTimeOfDay(endDate)}
```

Comment:
shows as two lines with icon positioned in between the lines vertically on my screen - can we organise the data in a way so that it is one line 
i believe my macbook is 1440x900 for your ref